app.py

```
@@ -40,7 +44,13 @@ def get_titles():
     # print("get_titles:")
     # print(titles)
 
+    try:
+        from make2 import event
+    except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)